example-jobs/envTestJob.groovy

```diff
@@ -2,7 +2,7 @@
 import static jenkins.automation.utils.EnvironmentUtils.isDev
 
 
-def env=${ENVIRONMENT}
+def env="${ENVIRONMENT}"
 if (isDev(env)){
```

Comment:
@marcesher You are right.
@vickumar1981 long story about groovy script vs. class context and bindings, but in short: "No. We can't."  We can chat more about it if you interested in the mechanics.